### PR TITLE
finished implementation for /dashboard

### PR DESCRIPTION
**Pull request by Tratori**
Splitting ffeature/SC-3560 up for better legibility
(I at least hope so)

- Links zu verwandten PRs anderer Repositories
  - https://github.com/schul-cloud/schulcloud-client/pull/1702
- Link zum Ticket https://ticketsystem.schul-cloud.org/browse/SC-3560

### CODE REVIEW

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5313026&bid=17389212)

File: locales/de.json [hunk 1]
```diff
@@ -1,524 +1,545 @@
...
+    },
+    "addons": {
+        "headline": {
+            "addons": "Add-ons",
+            "portfolio": "Portfolio"
+        },
+        "text": {
+            "portfolioDescription": "Erzeuge dein digitales Portfolio in einem flexiblen persönlichen Lernsystem. Teile die Erfolge und deine Entwicklung an einem Ort, den du selbst gestaltest. Arbeite mit anderen in Gruppen zusammen."
+        }
+    },
```
Comment (adrianjost) [inline locales/de.json:47]:
```suggestion
```

File: controllers/dashboard.js [hunk 2]
```diff
@@ -160,6 +160,7 @@ router.get('/', (req, res, next) => {
 			return [];
 		});
 
+
```
Comment (adrianjost) [inline controllers/dashboard.js:163]:
```suggestion
```

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5313026&bid=17389212)

**Pull request merged**